src/sql/workbench/contrib/notebook/browser/cellViews/code.component.ts

```diff
@@ -277,6 +277,15 @@ export class CodeComponent extends CellView implements OnInit, OnChanges {
 				this.setFocusAndScroll();
 			}
 		}));
+		this._register(this.cellModel.onExecutionComplete(() => {
+			// Allow screen reader to announce when a cell is done running
+			let element: Element = document.getElementsByClassName('sr-only')[0];
+			element.innerHTML = 'Cell execution is complete';
```

Comment:
1. This should be localized
2. You should use the [alert](https://github.com/microsoft/azuredatastudio/blob/2bc6a0cd012de69504061a876070cfa43533fedc/src/vs/base/browser/ui/aria/aria.ts#L49) function instead of making your own

Reply:
@lucyzhang929  the execute button for the cell will become disabled and a spinner will be displayed, I think you can add the call to alert function in there instead of having another event subscriber.

Reply:
Didn't realize there was an alert function, thanks! Also updated where I put the alert so I don't need a new event.